submissions/evgenii-del/oop-exercise/index.js

```diff
@@ -0,0 +1,96 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/evgenii-del/a-tiny-JS-world
+   Web app: http://crooked-discovery.surge.sh/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+    constructor(species, name, gender, saying) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.saying = saying;
+    }
+
+    toString(object) {
+        return [this.species, this.name, this.gender, this.saying].join(';');
+    }
+}
+
+class Man extends Inhabitant {
+    constructor(name, gender, saying = "Hi!", hands = 2, legs = 2) {
```

Comment:
Doesn't `Man` imply gender? Yet unused variable.
Please check your code for code optimizations when refactoring.
Here we've got quite small amount of code. Tech debt may affect code scalability.